packages/typespec-ts/src/modular/buildCodeModel.ts

```diff
@@ -1318,9 +1319,38 @@ function emitOperationGroups(
       operations: clientOperations
     });
   }
+  resolveConflictIfExist(operationGroups);
   return operationGroups;
 }
 
+function resolveConflictIfExist(operationGroups: OperationGroup[]) {
```

Comment:
Detect if there is name conflict cross operation groups, if yes, add operation group name prefix.

Reply:
Can you add this check for RLC as well ? thx.

Reply:
Done.